webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
Mid-air collision, my reply was to David. Silvia's wording also WFM, but "these words do not appear in all uppercase letters in this specification."